### ISSUE TITLE
[Bug]: Player timeout/freezes trying to connect to a server they just disconnected from

### ISSUE THREAD

**Issue by OzTheM9** [AllTheMods/ATM-9#1684]
### Possible Fixes

Yes

### Modpack Version

0.2.58

### What happened?

We're running into the issue that a lot of our players cannot reconnect after losing connection to our server. We've tried increasing the player timeout value to 120, but that hasn't helped. The issue is very inconsistent, but a lot of people are having it across a lot of different computers. After restarting the client its possible to connect just fine.

**Comment by cstalker** [comment]
this happend to me and i had to delete and re install the pack and it fixed it. extrema but it worked.

**Comment by YiRanMushroom** [comment]
Any idea on how to fix this easier? I cannot reinstall the modpack every time. Is there a better solution?

**Comment by x13x54** [comment]
> Any idea on how to fix this easier? I cannot reinstall the modpack every time. Is there a better solution?

yes, though only slightly better for now: restarting your game every time this happens :)
